### DISCUSSION TITLE
【灌水】新人越来越少了

### DISCUSSION THREAD

**Discussion by March-777th** [share121/inter-knot#5328] [灌水]
![1726876775580](https://github.com/user-attachments/assets/9f9924cf-2dca-4e86-964d-4de5f17d24a7)
什么时候引波大的<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/40f465ef5312e4ba0b19c596b9999cef_8760502457253372865.png" alt="三月七 无语" width="75" >

**Comment by RWHwaxili** [comment]
软件通过局里备案上架到应用市场就可以了

**Comment by Lephaethon** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >引波大的就要变混乱力

**Comment by aino2333** [comment]
不要引大的，会变得不幸（）

**Comment by aomengxin** [comment]
这是咱的小角落（）

**Comment by fzdqz** [comment]
等到1.4zzz正式开服_(•̀ω•́ 」∠)_

**Comment by TNCNET** [comment]
1.人多的社区，肯定是乱的。这个显而易见。你不知道人多了会怎么样。

2.软件毕竟没过审核，这么做可能会被注意到，然后被夹（软件必须审批后上架）

3.有一些门槛，但是也有好处：
•知道没法直连GitHub这些网站的时候该怎么办；
•能正确的打开/注册/登录和使用GitHub；
•会绑定设备码，配置软件，等等。
•有问题能正确，准确的提问，并且给出相关信息。

所以能筛选用户，也算是还原了原作（确信）。

**Comment by ASTOU2311** [comment]
hi,有人知道为什么打开不是绳网的画面而是gi
![image](https://github.com/user-attachments/assets/ea85ea96-cf44-4432-880f-80b704e4346a)
该如何查看显示的图片？

**Reply by March-777th** [reply]
可以简单的把绳网理解为github讨论区的一个皮肤<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
至于查看图片，进入网页后等待图片加载玩，直接点击就行

**Comment by LeoneMafia** [comment]
说真的，感觉圈子小点也不是坏事，林子大了真的是什么鸟都会有，我想这样总比让一堆不明生物搅得乌烟瘴气要好

**Comment by 3578903680** [comment]
划水ing
